apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -31,17 +30,34 @@ describe('Button Accessibility Testing', () => {
   });
 
   it('Button - Validate accessibilityRole is correct', async () => {
-    await expect(await ButtonPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+    await expect(
+      await ButtonPageObject.compareAttribute(ButtonPageObject._primaryComponent, Attribute.AccessibilityRole, BUTTON_A11Y_ROLE),
+    ).toBeTrue();
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Button - Set accessibilityLabel', async () => {
```

Comment:
nit I think we should follow the same wording as the rest of the spec files i.e. "Set accessibilityLabel -> Validate accessibilityLabel is correct"